mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/logging/impl/Log.java

```diff
@@ -88,8 +88,8 @@ public interface Log extends BasicLogger {
 	void indexingProgressRaw(long doneCount, long elapsedMs);
 
 	@LogMessage(level = INFO)
-	@Message(id = ID_OFFSET_LEGACY_ENGINE + 31, value = "Mass indexing progress: %2$.2f%% [%1$f documents/second].")
-	void indexingProgressStats(float estimateSpeed, float estimatePercentileComplete);
+	@Message(id = ID_OFFSET_LEGACY_ENGINE + 31, value = "Mass indexing progress: %3$.2f%% [%1$f documents/second [current] / %2$f documents/second [since start]].")
```

Comment:
Ah, I didn't realize we already had square brackets in this message... it looks a bit weird now, doesn't it?

Maybe something like this?

```suggestion
	@Message(id = ID_OFFSET_LEGACY_ENGINE + 31, value = "Mass indexing progress: %3$.2f%% [%1$f documents/second since last message, %2$f documents/second since start].")
```